Userland/Libraries/LibCore/FilePermissionsMask.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021, Xavier Defrang <xavier.defrang@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Error.h>
+#include <AK/String.h>
+#include <sys/stat.h>
+
+namespace Core {
+
+class FilePermissionsMask {
+private:
+    mode_t m_clear_mask; // the bits that will be cleared
+    mode_t m_write_mask; // the bits that will be set
```

Comment:
We use public, protected, private as the usual order of class members and functions